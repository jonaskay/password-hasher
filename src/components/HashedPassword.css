.HashedPassword {
  text-align: center;
}

.HashedPassword > input {
  border: none;
  border-radius: .125rem;
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.12), 0 2px 4px 0 rgba(0,0,0,0.08);
  box-sizing: border-box;
  font-family: Menlo, Monaco, Consolas, Liberation Mono, Courier New, monospace;
  font-size: 1.125rem;
  padding: .75rem .5rem;
  text-align: center;
  width: 100%;
}
