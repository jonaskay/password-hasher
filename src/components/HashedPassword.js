import React, { Component } from 'react';
import './HashedPassword.css';

class HashedPassword extends Component {
  render() {
    return (
      <div className="HashedPassword">
        <input type="text" value="4QAIn8SvaW" />
      </div>
    );
  }
}

export default HashedPassword;
