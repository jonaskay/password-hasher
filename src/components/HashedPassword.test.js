import React from 'react';
import { shallow } from 'enzyme';
import HashedPassword from './HashedPassword';

describe('HashedPassword', () => {
  it('renders correctly', () => {
    expect(shallow(<HashedPassword />)).toMatchSnapshot();
  });
});
