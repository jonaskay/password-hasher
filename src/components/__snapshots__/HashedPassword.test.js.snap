// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HashedPassword renders correctly 1`] = `
<div
  className="HashedPassword"
>
  <input
    type="text"
    value="4QAIn8SvaW"
  />
</div>
`;
